test/new-e2e/system-probe/test-runner/main.go

```diff
@@ -63,9 +63,11 @@ var timeouts = map[*regexp.Regexp]time.Duration{
 	regexp.MustCompile("pkg/network/usm$"):            30 * time.Minute,
 }
 
+// Note: security tests don't have the full path to the test, so increase the timeout as a temporary solution
```

Comment:
What does the path have to do with the timeout?

Reply:
I'm not sure why, but the security tests do not retain the package name in the tests: see [this in security-agent](https://gitlab.ddbuild.io/DataDog/datadog-agent/-/jobs/418377569#L247) vs [this in system-probe](https://gitlab.ddbuild.io/DataDog/datadog-agent/-/jobs/418670099#L2172): in system-probe we have `pkg/network/usm`, in security it's just `tests`. Maybe @paulcacheux can help here?

Reply:
Let's figure that out, because I don't want to increase the default timeout.

Reply:
I'm not sure why this is the case, if I look at current kitchen tests https://gitlab.ddbuild.io/DataDog/datadog-agent/-/jobs/418377757, 
```
 [4.14.262-200.489.amzn2.x86_64][d] === SKIP: github.com/DataDog/datadog-agent/pkg/security/tests TestOpen/io_uring (0.25s)
```
so it has the fullpath info

Reply:
Solved, it was the filesystem path where the tests were placed.